### PR TITLE
Send report through ActionJob

### PR DESCRIPTION
**Pull request by dLobatog**
This allows us to have Sidekiq workers without sharing any volumes. The
report is send as a string over to Redis, serialized as an ActiveJob
job.

### CODE REVIEW

File: app/jobs/parse_report_job.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def perform(file, account, b64_identity)
     parser = XCCDFReportParser.new(file, account, b64_identity)
     parser.save_rule_results
   ensure
-    File.delete(file)
+    File.delete(parser.report_path) if File.exist? parser.report_path
```
Comment (dLobatog) [inline app/jobs/parse_report_job.rb]: This shouldn't really get called as `Tempfile` destroys the file as soon as it's not reclaimed by the context, at garbage collection time, which is why I'm ensuring the file exists before removing it

File: test/controllers/systems_controller_test.rb [hunk 2]
```diff
@@ -16,10 +16,11 @@ class SystemsControllerTest < ActionDispatch::IntegrationTest
 
   test 'index accepts search' do
     scope = mock('scope')
-    SystemsController.any_instance.expects(:policy_scope).with(Host).returns(scope)
+    SystemsController.any_instance.expects(:policy_scope).with(Host)
+                     .returns(scope)
     scope.expects(:search_for).with('foo=bar')
 
-    get systems_url, params: {search: 'foo=bar'}
+    get systems_url, params: { search: 'foo=bar' }
```
Comment (akofink) [inline test/controllers/systems_controller_test.rb]: I'm really not a fan of this kind of thing in PRs, unless you put it in a separate commit. It's just noise in the review. If you'd like to enforce some of these style things like line length, etc., let's update the rubocop config! I'm fine to keep the changes in this one PR, though.

Comment (dLobatog) [reply test/controllers/systems_controller_test.rb]: Tests didn't pass if I didn't add this apparently :disappointed:
